examples/service-fastapi/app/service.py

```diff
@@ -51,7 +51,7 @@ async def me(user: User = Depends(get_current_user)):
 
 
 @router.get("/debug")
-async def index(request: Request, user: User = Depends(get_current_user)):
+async def index(request: Request, user: User = Depends(get_current_user)):  # noqa
```

Comment:
This one could be called `debug` for the page name. It's not technicaly a _bug_ that two handler functions have the same name, but it's not intentional and probably due to a copy/paste.